packages/styled-components/src/utils/hasher.js

```diff
@@ -2,36 +2,15 @@
 
 import generateAlphabeticName from './generateAlphabeticName';
 
-function murmurhash(c: string): number {
-  for (var e = c.length | 0, a = e | 0, d = 0, b; e >= 4; ) {
-    (b =
-      (c.charCodeAt(d) & 255) |
-      ((c.charCodeAt(++d) & 255) << 8) |
-      ((c.charCodeAt(++d) & 255) << 16) |
-      ((c.charCodeAt(++d) & 255) << 24)),
-      (b = 1540483477 * (b & 65535) + (((1540483477 * (b >>> 16)) & 65535) << 16)),
-      (b ^= b >>> 24),
-      (b = 1540483477 * (b & 65535) + (((1540483477 * (b >>> 16)) & 65535) << 16)),
-      (a = (1540483477 * (a & 65535) + (((1540483477 * (a >>> 16)) & 65535) << 16)) ^ b),
-      (e -= 4),
-      ++d;
-  }
-  switch (e) {
-    case 3:
-      a ^= (c.charCodeAt(d + 2) & 255) << 16;
-    case 2:
-      a ^= (c.charCodeAt(d + 1) & 255) << 8;
-    case 1:
-      (a ^= c.charCodeAt(d) & 255),
-        (a = 1540483477 * (a & 65535) + (((1540483477 * (a >>> 16)) & 65535) << 16));
-  }
-  a ^= a >>> 13;
-  a = 1540483477 * (a & 65535) + (((1540483477 * (a >>> 16)) & 65535) << 16);
-  return (a ^ (a >>> 15)) >>> 0;
-}
+export const hash = (x: string): number => {
+  /* prettier-ignore */
+  for (var h = 5381 | 0, i = 0, l = x.length | 0; i < l; i++)
```

Comment:
what's this one?

Reply:
That's djb2 which has acceptably low collision rates even compared with murmurhash, is also not a cryptographic hasher, and quite short https://gist.github.com/MohamedTaha98/ccdf734f13299efb73ff0b12f7ce429f